### PR TITLE
Update from Weblate

### PR DESCRIPTION
**Pull request by xwikiorgci**

### CODE REVIEW

File: application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarMacroTranslations.de.xml [hunk 1]
```diff
@@ -39,24 +39,24 @@
...
-rendering.macro.moccacalendar.parameter.calendarDoc.name=Kalenderdokument
-rendering.macro.moccacalendar.parameter.calendarDoc.description=Das Kalender-Dokument, dessen Termine angezeigt werden (wird nur verwendet, wenn im obigen Wert zur Anzeige der Termine 'page' verwendet wird. Falls leer, wird das aktuelle Dokument verwendet. Wenn das Dokument kein Kalender ist, werden keine Termine angezeigt.
+rendering.macro.moccacalendar.parameter.filter.description=Mögliche Werte: 'wiki','space' oder 'page', wobei 'page' nur Termine eines Kalenders anzeigt, 'space' alle Termine in Unterseiten anzeigt, und 'wiki' alle Termine im Wiki anzeigt.
+rendering.macro.moccacalendar.parameter.calendarDoc.name=Kalenderseite
+rendering.macro.moccacalendar.parameter.calendarDoc.description=Die Kalender-Seite, deren Termine angezeigt werden (wird nur verwendet, wenn im obigen Wert zur Anzeige der Termine 'page' verwendet wird. Falls leer, wird die aktuelle Seite verwendet. Wenn ddie Seite kein Kalender ist, werden keine Termine angezeigt.
```
Comment (ClemensRobbenhaar) [inline application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarMacroTranslations.de.xml]: Ouch, typo: `Wenn ddie Seite ...`

Comment (surli) [reply application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarMacroTranslations.de.xml]: I let you edit the translation on weblate, not merging right now :)

Comment (ClemensRobbenhaar) [reply application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarMacroTranslations.de.xml]: It should be fixed by now on weblate. :)

Comment (surli) [reply application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarMacroTranslations.de.xml]: FTR: I keep it like that for now, still trying to understand why weblate doesn't commit automatically.

Comment (ClemensRobbenhaar) [reply application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarMacroTranslations.de.xml]: Ok. Is weblate expected to augment the existing pull request, or should it create a new one?  
Anyway, as I do not know how to help with debugging this I will just wait and see. :)

Comment (surli) [reply application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarMacroTranslations.de.xml]: > Is weblate expected to augment the existing pull request

it augments them: basically it push forces them

Comment (ClemensRobbenhaar) [reply application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarMacroTranslations.de.xml]: The may the force be with weblate back soon.

**Comment by surli** [comment]
@ClemensRobbenhaar 	I let you check and merge, it should be ok now

**Comment by ClemensRobbenhaar** [comment]
Yes, looks good now, thanks!

**Pull request merged**